src/Resources/contao/controllers/BackendIndex.php

```diff
@@ -71,4 +73,32 @@ public function run()
 
 		return $objTemplate->getResponse();
 	}
+
+	/**
+	 * Gets the security messages for the login view.
+	 *
+	 * @return string
+	 */
+	private function getLoginMessages()
+	{
+		$strMessages = \Message::generateUnwrapped();
+
+		$exception = \System::getContainer()->get('security.authentication_utils')->getLastAuthenticationError();
+
+		if ($exception instanceof LockedException)
+		{
+			$strMessages .= '<p class="tl_error">' . sprintf($GLOBALS['TL_LANG']['ERR']['accountLocked'], $exception->getLockedMinutes()) . '</p>';
```

Comment:
Why not `\Message::addError()`?

Reply:
it would also render other error messages. The only advantage would be the "automatic" markup, right?

Reply:
If there are other error messages, they should probably be rendered, don't you think?

Reply:
I'm not really sure, because it's the backend login screen and you're probably not supposed to see them if you're not logged in? Though that has been the case already up until now …

Reply:
It has. And also, if you are not logged in, there will be no other messages.

Reply:
Changed in 596453b420c72e7192e0712a955b638fe3e3dd63